src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java

```diff
@@ -37,6 +37,7 @@ public static void main(String[] args) {
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
+        return depositAmount+depositAmount*depositAnnualPercentage/100*depositYears;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"